docs/pages/reference/access-controls/roles.mdx

```diff
@@ -142,6 +144,55 @@ spec:
       'workload': ['database', 'backup']
 ```
 
+### `client_idle_timeout`
+
+This example configures a global idle timeout of **30 minutes**. After 30 minutes of inactivity, the session will be terminated.
```

Comment:
I'd be careful simplifying _client idle timeout_ to _idle timeout_.

_Client_ idle timeout refers to there being no client activity. Idle timeouts in general often just mean no traffic whatsoever.